docs/content/docs/recipes/angular.md

```diff
@@ -1,6 +1,276 @@
+      const id = await container.attach();
+      location.hash = id;
+    }
+    else {
+      ({ container } = await client.getContainer(containerId, containerSchema));
+    }
+    ```
+
+1. Replace `TODO 3` with the following code.
+
+    ```js
+    return container.initialObjects.sharedTimestamp as SharedMap;
+    ```
+
+### Keep the Fluid data synchronized
+
+To ensure that both local and remote changes to the timestamp are reflected in the UI, we will use the `localTimestamp` component property to store the local timestamp value and ensure that it is updated whenever any client changes the `fluidSharedObjects` value.
+
+1. Below the preceding `getFluidData` function add the following code.
+
+    ```js
+    syncData() {
+      // Only sync if the Fluid SharedMap object is defined.
+      if (this.sharedTimestamp) {
+        // TODO 4: Set the value of the localTimestamp object that will appear in the UI.
+
+        // TODO 5: Register handlers.
+      }
+    }
+    ```
+
+1. Replace `TODO 4` with the following code. Note about this code:
+
+    - The Fluid `SharedObject.get` method returns the data of the `SharedObject` (in this case the `SharedMap` object), which is roughly the `SharedObject` without any of its methods. So the `updateLocalTimestamp` function is setting the `localTimestamp` property to a copy of the data of the `SharedMap` object. (The key "time" that is passed to `SharedObject.get` is created in a later step. It will have been set by the time this code runs the first time.)
+    - `updateLocalTimestamp` is called immediately to ensure that `localTimestamp` is initialized with the current shared timestamp value.
+
+    ```js
+    this.updateLocalTimestamp = () => { this.localTimestamp = { time: this.sharedTimestamp!.get("time") } };
+    this.updateLocalTimestamp();
+    ```
+
+1. To ensure that the `localTimestamp` state is updated whenever the `sharedTimestamp` is changed *even by other clients*, replace `TODO 5` with the following code. Note that because `updateLocalTimestamp` calls the state-setting function `setTimestamp`, a rerender is triggered whenever any client changes the Fluid `sharedTimestamp`.
+
+    ```js
+    this.sharedTimestamp!.on('valueChanged', this.updateLocalTimestamp!);
+    ```
+
+1. It is a good practice to deregister event handlers when the Angular component dismounts, so add the following code to the `ngOnDestroy` function we previously defined.
+
+    ```js
+    // Delete handler registration when the Angular App component is dismounted.
+    this.sharedTimestamp!.off('valueChanged', this.updateLocalTimestamp!);
+
+Now that we've defined how to get and synchronize our Fluid data, we need to tell Angular to call `getFluidData` and `syncData` when the application starts up and then store the result in component properties. So add the following code to the `ngOnInit` function we defined previously.
+
+```js
+this.sharedTimestamp = await this.getFluidData();
+this.syncData();
+
```

Comment:
Nit: missing closing ticks

```suggestion
    ```js
    this.sharedTimestamp = await this.getFluidData();
    this.syncData();
    ```

```